web/styles/popovers.css

```diff
@@ -161,33 +161,44 @@
     }
 }
 
-.user-card-popover-actions .custom_user_field {
+.custom-profile-field-value {
```

Comment:
These selectors should probably still be scoped inside something about popovers; these presentation styles are not applicable to every custom profile field value we might display anywhere in the app.